spec-insert/lib/renderers/body_parameters.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require_relative 'components/base_mustache_renderer'
+require_relative 'components/parameter_table_renderer'
+
+# Renders request body parameters
+class BodyParameters < BaseMustacheRenderer
+  self.template_file = "#{__dir__}/templates/body_parameters.mustache"
+
+  def initialize(action, args, is_request:)
+    super(action, args)
+    @is_request = is_request
+    @body = is_request ? @action.request_body : @action.response_body
+    @params_group = @body.params_group
+  end
+
+  def header
+    @header ||= "#{@is_request ? 'Request' : 'Response'} body"
```

Comment:
Would this fix the expected output error?

```suggestion
    @header ||= "#{@is_request ? 'Request' : 'Response'} body fields"
```